artview/components/plot.py

```diff
@@ -112,23 +103,26 @@ def __init__(self, Vradar, Vfield, Vtilt, Vlims=None, \
         self.cm_names = [m for m in cm.datad if not m.endswith("_r")]
         self.cm_names.sort()
 
-        # Create a figure for output
-        self._set_fig_ax(nrows=1, ncols=1)
-
         # Create tool dictionary
         self.tools = {}
-        self.tools['zoompan'] = None
+
+        # Check the file type and initialize limts
+        # Create a figure for output
+        self._check_file_type()
```

Comment:
The _initialize limits need to be set in three circumstances:
1. before drawing the initial plot 
2. when the plot type (i.e. ppi goes to rhi) changes
3. when the reset default is chosen.

This causes a "hard" reset to a default state.
Maybe this could be broken into a couple of different functions. The reason the figure needs it is to define the aspect ratio of the plot (RHI and Airborne both have longer X than Y axes). This was convenient a while back, but could probably be changed now.